services/userService.js

```diff
@@ -17,25 +17,24 @@ const signUp = async (email, name, password, address, phone, birthdate) => {
     birthdate
   );
 };
-
+//로그인
 const signIn = async (email, password) => {
   const user = await userDao.getUserByEmail(email);
-  console.log(user);
 
-  if (user.length == 0) {
+  if (user == undefined) {
```

Comment:
```suggestion
  if (!user) {
```
위와 같이 수정되어도 똑같이 동작합니다.

Reply:
네 수정했습니다 !